src/js/letters/actions/letters.js

```diff
@@ -1,16 +1,52 @@
-import { apiRequest } from '../utils/helpers';
+import Raven from 'raven-js';
+
+import { apiRequest } from '../utils/helpers.jsx';
+import {
+  BACKEND_AUTHENTICATION_ERROR,
+  BACKEND_SERVICE_ERROR,
+  INVALID_ADDRESS_PROPERTY,
+  GET_LETTERS_FAILURE,
+  GET_LETTERS_SUCCESS,
+  GET_BENEFIT_SUMMARY_OPTIONS_FAILURE,
+  GET_BENEFIT_SUMMARY_OPTIONS_SUCCESS,
+  GET_LETTER_PDF_FAILURE,
+  GET_LETTER_PDF_SUCCESS,
+  UPDATE_BENFIT_SUMMARY_REQUEST_OPTION
+} from '../utils/constants';
 
 export function getLetterList() {
   return (dispatch) => {
     apiRequest(
       '/v0/letters',
       null,
       response => dispatch({
-        type: 'GET_LETTERS_SUCCESS',
+        type: GET_LETTERS_SUCCESS,
         data: response,
       }),
-      () => dispatch({ type: 'GET_LETTERS_FAILURE' })
-    );
+      (response) => {
+        const error = response.errors.length > 0 ? response.errors[0] : undefined;
+        if (error) {
+          if (error.status === '503' || error.status === '504') {
+            // Either EVSS or a partner service is down or EVSS times out
+            return dispatch({ type: BACKEND_SERVICE_ERROR });
+          }
+          if (error.status === '403') {
+            // Backend authentication problem
+            return dispatch({ type: BACKEND_AUTHENTICATION_ERROR });
+          }
+          if (error.status === '422') {
+            // Something about the address is invalid, unable to process the request
+            return dispatch({ type: INVALID_ADDRESS_PROPERTY });
+          }
+        }
+        return Promise.reject(
+          new Error('letters getLetterList() received unexpected status code: `${response.status}`')
```

Comment:
Good question. I think you're right. Confirming with @raquelromano-gov because I copied this from some code she wrote for GIBS, but in testing this I don't think there is such a thing as `response.status`; it seems like this is a holdover from when we thought the status would be returned directly in the response as `response.status`, but then realized we're actually getting an errors array in the response, and the status is within that.